core/shared/src/main/scala/zio/RIO.scala

```diff
@@ -512,6 +512,22 @@ object RIO {
   )(f: A => RIO[R, B])(implicit ev: CanFail[Throwable]): RIO[Nothing, (List[Throwable], List[B])] =
     ZIO.partitionM(in)(f)
 
+  /**
+   * @see See [[zio.ZIO.partitionMPar]]
+   */
+  final def partitionMPar[R, A, B](
+    in: Iterable[A]
+  )(f: A => RIO[R, B])(implicit ev: CanFail[Throwable]): RIO[Nothing, (List[Throwable], List[B])] =
```

Comment:
It occurs to me we can delete all these implicit `CanFail[Throwable]`, because, well, `Throwable` can fail. 😄  But maybe best as a separate PR since I see the other methods too.

Reply:
This made me laugh. Thank you for that. I can do that straight after this one.